usr/src/uts/common/fs/zfs/sys/dmu_objset.h

```diff
@@ -121,11 +121,14 @@ struct objset {
 
 	/* Protected by os_lock */
 	kmutex_t os_lock;
-	list_t os_dirty_dnodes[TXG_SIZE];
-	list_t os_free_dnodes[TXG_SIZE];
+	struct objset *os_origin_mooch_objset;
```

Comment:
Can't seem to find the meaning of this. Maybe a leftover from earlier revisions?